core/templates/dev/head/expressions/expression-parser.service.ts

```diff
@@ -0,0 +1,16 @@
+import { downgradeInjectable } from "@angular/upgrade/static";
```

Comment:
Please use single quotes instead of double quotes. Actually this quote changes should have been blocked by eslint in the pre push flow, shouldn't it? @kevinlee12 @seanlip

Reply:
Hm, yes, I think it should have. I'm not sure why not. @Hudda @kevinlee12 do you know?

Reply:
The linter does have that toggled though the linter has timed out before that error could surface. Funny enough, the Github Actions linter didn't run on this either, can you merge from the latest develop?

Reply:
@kevinlee12 I merged from latest develop, but nothing happened. I also ran the pre_commit_linter locally and it did not throw the error.

@seanlip As seen in the screenshot, the pre_commit_linter runs in 26 minutes on my laptop now, which is more than the 10 minute limit, but it does complete successfully. Would it be okay to increase the timeout in the test, or is this too long? I also noticed that the command run in the test, `python -m scripts.pre_commit_linter --path=. --verbose`, lints all the files in the oppia directory. Wouldn't it be sufficient to only lint the files touched in the current PR? 

![image](https://user-images.githubusercontent.com/6963537/73605249-ef7c3a00-4569-11ea-8857-a43d93ebbefa.png)

Reply:
Hi @tyleri, I thought that the pre-commit linter, when run on your local machine, should only look at the files which have been touched. If this is not the case, something seems to be wrong. Just to clarify, are you referring to the linter when run on your local machine, or the linter when run on travis/circleCI?

Reply:
Sorry, I am talking about the linter when run on travis/circleCI. You are correct that when it runs locally, it only looks at the touched files. I was intentionally running the command on all the files (like what travis/circleCI does) to see how long my machine would take to lint all the files. As seen in the screenshot above, it took 26 minutes on my machine, so it's understandable that travis/circleCI is timing out with a 10 minute limit.

Reply:
Thanks for clarifying! 

Could you talk more about this "10 minute limit"? On other PRs, the lint checks seem to pass for the ones I looked at  (e.g. https://circleci.com/gh/oppia/oppia/32908) and the log indicates that they ran to completion (in 28 minutes or so). I think the 10-minute limit _might_ be "10 minutes with no activity", rather than "10 minutes for the whole run", but I am not completely sure.

That said, given that this issue isn't happening on other PRs, I'm a bit confused about what's different about this PR in particular...

Reply:
The 10-minute limit being for "10 minutes with no activity" makes a lot more sense. In that case, I am not quite sure what is different about this PR, as I am only changing a few files that do not seem related to the linting (except for the change I made to the pre_commit_linter to try to fix this issue). I will continue to try merging from develop and look into this.